src/widget/tabs/preview-tab.js

```diff
@@ -174,13 +174,20 @@ class PreviewTab extends BasePreviewTab {
       URL.revokeObjectURL(src)
       return df.reject()
     }).done(() => {
-      var videoTag
+      if (
+        file.state() !== 'pending' ||
```

Comment:
Why pending? What if file as resolved?

Reply:
if the file is resolved everything will be ok
I chose `resolve` state because that state used in case with image here: https://github.com/uploadcare/uploadcare-widget/blob/e4f1d9585d08411e73f8e1a5f455bad35f8be219/src/widget/tabs/preview-tab.js#L136-L146

and we didn't have problems like that with images before